src/frontend/src/pages/PIADetailPage/index.tsx

```diff
@@ -137,7 +152,39 @@ const PIADetailPage = () => {
     }
   };
 
+  const handleSubmit = () => {
+    handleShowModal('submit');
+  };
+
+  const handleStatusSubmit = async () => {
+    const requestBody: Partial<IPIAIntake> = {
+      status:statusLocal,
+    }; 
+    try {
+      const res = await HttpRequest.patch<IPIAResult>(
+        API_ROUTES.PATCH_PIA_INTAKE.replace(':id', `${id}`),
+        requestBody,
+      );
+        /* PIA will be set after data is updated in backend */
+        pia.status = statusLocal;
+        setPia({...pia, status:statusLocal});
```

Comment:
do we need to do this at all?

Reply:
We use this to trigger the call to fetch data from the backend using useeffect.

Reply:
line 169 is not needed.